Andy Solar System Test/Assets/Planets.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/// Sample Code for CS 491 Virtual And Augmented Reality Course - Fall 2017
 /// written by Andy Johnson
-///
+/// 
```

Comment:
This file too is all whitespace edits.  Don't commit files that you didn't work on.